features-json/svg.json

```diff
@@ -49,21 +49,21 @@
   "stats":{
     "ie":{
       "5.5":"n",
-      "6":"p",
-      "7":"p",
-      "8":"p",
-      "9":"y #2",
-      "10":"y #2",
-      "11":"y #2"
+      "6":"n",
+      "7":"n",
+      "8":"n",
+      "9":"p #2 #3",
+      "10":"p #2 #3",
+      "11":"p #2 #3"
```

Comment:
Don't think you meant to change `y` to `p` here? (since `p` refers to "supported with polyfill")

Reply:
Ah yes off course! I had no idea that that existed, confused it with `a` - `Partial Support`